File: app/src/main/java/com/termux/zerocore/utils/SaveData.java
package com.termux.zerocore.utils;

import android.content.Context;
import android.content.SharedPreferences;

import com.example.xh_lib.utils.UUtils;


public class SaveData {


    public static void saveData(String key, String values) {


        SharedPreferences xinhao = UUtils.getContext().getSharedPreferences("xinhao", Context.MODE_PRIVATE);

        xinhao.edit().putString(key, values).apply();


    }


    public static String getData(String key) {


        SharedPreferences xinhao = UUtils.getContext().getSharedPreferences("xinhao", Context.MODE_PRIVATE);

        String def = xinhao.getString(key, "def");

        return def;

    }

}

File: xh_lib/src/main/java/com/example/xh_lib/utils/SaveData.kt
package com.example.xh_lib.utils

import android.content.Context
import com.google.gson.Gson

/**
 * @author ZEL
 * @create By ZEL on 2020/7/29 11:35
 **/
object SaveData {

    fun saveStringOther(key: String, values: String) {

        val sharedPreferences = UUtils.getContext().getSharedPreferences("start_flash", Context.MODE_PRIVATE)

        sharedPreferences.edit().putString(key, values).apply()

    }

    fun getStringOther(key: String): String? {

        val sharedPreferences = UUtils.getContext().getSharedPreferences("start_flash", Context.MODE_PRIVATE)

        return sharedPreferences.getString(key, "def")

        // return ""

    }



}